Makefile

```diff
@@ -1,9 +1,17 @@
 NAME    := unsee
 VERSION := $(shell git describe --tags --always --dirty='-dev')
+GO      := GO15VENDOREXPERIMENT=1 go
```

Comment:
Go >=1.8 is required to compile unsee, there's no point in setting `GO15VENDOREXPERIMENT`